sketch/payload_function.js

```diff
@@ -0,0 +1,35 @@
+function Decoder(bytes, port) {
+
+  var decoded = {};
+
+  bssid = shiftBytes(bytes);
+  decoded.Bssid1 = bssid.join('');
+
+  bssid = shiftBytes(bytes);
+  decoded.Bssid2 = bssid.join('');
+
+  bssid = shiftBytes(bytes);
+  decoded.Bssid3 = bssid.join('');
+
+  return decoded;
+}
+
+function pad(n) {
+  return (n < 10 || n == 'a' || n == 'b' || n == 'c' || n == 'd' || n == 'e' || n == 'f') ? ("0" + n) : n;
```

Comment:
You can compare against chars as well, e.g. `n >= 'a' && `n <= 'f'`

Also use consistent quotes, don't mix `'` and `"`

Reply:
I did have it like that initially, but when i compare them like that. the output looks like this:
![image](https://user-images.githubusercontent.com/24267554/33620924-f02d173e-d9f1-11e7-8d21-79d09f1ec334.png)